komga-webui/src/services/komga-series.service.ts

```diff
@@ -214,4 +214,52 @@ export default class KomgaSeriesService {
       throw new Error(msg)
     }
   }
+
+  async getThumbnails(seriesId: string): Promise<SeriesThumbnailDto[]> {
+    try {
+      return (await this.http.get(`${API_SERIES}/${seriesId}/thumbnails`)).data
+    } catch (e) {
+      let msg = `An error occurred while trying to retrieve thumbnails for series '${seriesId}'`
+      if (e.response.data.message) {
+        msg += `: ${e.response.data.message}`
+      }
+      throw new Error(msg)
+    }
+  }
+
+  async uploadThumbnail(seriesId: string, body: FormData) {
```

Comment:
building the `FormData` inside the function and exposing the 2 parameters would hide the underlying complexity in the service layer